templates/rhpam70-authoring.yaml

```diff
@@ -190,13 +190,14 @@ parameters:
   example: http://nexus.nexus-project.svc.cluster.local:8081/nexus/content/groups/public/
   required: false
 - displayName: Maven repository username
-  description: Username to access the Maven repository, if required.
```

Comment:
@ruromero : Why this change, both in the description and the behavior? This is incorrect. Credentials are in fact very rarely needed for accessing maven dependencies. Please open a jira as a major bug and revert this, making sure the original description is maintained and the field is optional across all templates.

CC @errantepiphany @spolti

Reply:
I will fix this in a follow-up PR.

Reply:
Fixed on https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/93 and https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/94